### DISCUSSION TITLE
破壞了bootstrap的原有樣式書寫方式，bv簡直糟糕透了

### DISCUSSION THREAD

**Discussion by Amado-github** [bootstrap-vue/bootstrap-vue#6796] [General]
在工作之餘學習一門前端框架是很耗費時間精力的，我實在沒有想到，bv竟然會改變bootstrap樣式的願有書寫方式，我不能在bv中復用我的bootstrap知識，在這方面來說，bv已經是一個失敗的作品了

**Comment by Amado-github** [comment]
糟糕透顶的作品

**Comment by lichaozhy** [comment]
那建议你分别使用vue和bootstrap来满足你的特定需求。

我个人在使用bv的时候，收获很大的益处，并大量复用了bootstrap的知识（布局、utils、和所有组件，以上这些均基于bootstrap的样式表）。BV面向vue所带来的新机遇，适度通过directive和props耦合了有关样式的编程逻辑，提供更加富有层次的声明式表达是很有意义的接口设计改进。

介于这种耦合和你所习惯的定制化要求之间，如果存在矛盾，必定可以抛弃这些实现，如你所熟悉的那样，去使用纯粹的HTML实现组件。而所谓“是否能够复用bootstrap的知识”，则是建立在一个特定的知识抽象层次之上才能讨论的事情。

bootstrap-vue，它既不是bootstrap，也不是vue。我个人建议避免发表这种完全没有必要的刻板偏见，“不使用”或“开发令自己满意的UI框架”通常是更有效的做法。
